src/main/java/org/jabref/gui/entryeditor/citationrelationtab/CitationRelationsTab.java

```diff
@@ -74,6 +87,9 @@ public class CitationRelationsTab extends EntryEditorTab {
     private final BibEntryRelationsRepository bibEntryRelationsRepository;
     private final CitationsRelationsTabViewModel citationsRelationsTabViewModel;
     private final DuplicateCheck duplicateCheck;
+    private EntryEditor entryEditor;
```

Comment:
Not used. Please remove.

It would be really nice if you scrolled through your source code in IntelliJ. That variable should be in light gray meaning: not used. 😅

The thing is that we as reviewers could overlook things. Moreover, if we comment on these kind things, there is no time to check the harder questions... 😇